framework/decode/vulkan_ascii_consumer_base.cpp

```diff
@@ -28,21 +29,40 @@
 GFXRECON_BEGIN_NAMESPACE(gfxrecon)
 GFXRECON_BEGIN_NAMESPACE(decode)
 
+/// The version of the JSON file format which will be produced.
+/// Inc the minor number when anything is added.
+#define GFXRECON_CONVERT_JSON_VERSION "0.8.0"
+
 VulkanAsciiConsumerBase::VulkanAsciiConsumerBase() : file_(nullptr) {}
 
 VulkanAsciiConsumerBase::~VulkanAsciiConsumerBase()
 {
     Destroy();
 }
 
-void VulkanAsciiConsumerBase::Initialize(FILE* file)
+void VulkanAsciiConsumerBase::Initialize(FILE*      file,
```

Comment:
There are lines like this from https://github.com/LunarG/gfxreconstruct/pull/830 showing as it hasn't merged to dev yet.